gulpfile.js

```diff
@@ -163,7 +163,7 @@ function buildCssPipeline(assetGroup, doConcat, doRebuild) {
         if (ext !== ".scss" && ext !== ".less" && ext !== ".css")
             throw "Input file '" + inputPath + "' is not of a valid type for output file '" + assetGroup.outputPath + "'.";
     });
-    var generateSourceMaps = assetGroup.hasOwnProperty("generateSourceMaps") ? assetGroup.generateSourceMaps : true;
+    var generateSourceMaps = assetGroup.hasOwnProperty("generateSourceMaps") ? assetGroup.generateSourceMaps : false;
```

Comment:
I couldn't find a way to prevent source maps from being different in CI than on my Windows PC. I think it can always be different between platforms, so if we save these files to the repo, then we shouldn't generate them.

Reply:
That's not what I've seen previously after fixing the ordering, nor now: https://github.com/OrchardCMS/OrchardCore/actions/runs/8726125446/job/23940643697 Even if the order is the same, as well as the line endings, there are some differences in the source maps for some reason.

![image](https://github.com/OrchardCMS/OrchardCore/assets/1976647/b26bcb82-c516-4f65-9d1f-e21d379b2cbe)

See this validation where I tried it only with a single CSS file: https://github.com/OrchardCMS/OrchardCore/actions/runs/8726486111. But this happens with all files.